README.md

```diff
@@ -19,129 +20,161 @@ If you already have the CLI installed and wish to upgrade to the latest version,
 pip install --upgrade cleanlab-studio
 ```
 
-## Workflow
+## Quickstart
 
-Uploading datasets to Cleanlab Studio is a two-step process.
+1. If this is your first time using the Cleanlab CLI, authenticate with `cleanlab login`. You can find your API key at <https://app.cleanlab.ai/upload>.
+
+2. Upload your dataset ([image](#upload-an-image-dataset), [text](#upload-a-text-dataset), or [tabular](#upload-a-tabular-dataset)) using `cleanlab dataset upload`.
 
-1. We generate a schema describing the dataset and its <a href="#data-types-and-feature-types">data and feature
-   types</a>, which is verified by the user.
-2. Based on this schema, the dataset is parsed and uploaded to Cleanlab Studio.
+3. Create a project in Cleanlab Studio and improve your dataset (e.g., correct some labels).
+
+4. [Download corrected data](#download-corrected-data) with `cleanlab cleanset download`.
 
 ### Upload a dataset
 
-To upload a dataset without
-first [generating a schema](https://github.com/cleanlab/cleanlab-studio/#generate-dataset-schema) (i.e. Cleanlab will
-suggest one for you):
+#### Upload an image dataset
 
-`cleanlab dataset upload -f [dataset filepath]`
+An image dataset consists of a collection of image files (organized in any way, in any folder hierarchy and with any file names), along with a metadata file specifying paths and labels (and optionally, other metadata). An image dataset might be organized like this:
 
-**Note:** For image datasets, `[dataset filepath]` refers to the path of the labels file. <a href="#dataset-format">Learn more about the labels file.</a>
+```
+- dogs
+  - scruffy.png
+  - spot.jpg
+- cats
+  - whiskers.png
+- fred.png
+- labels.csv
+```
 
-You will be asked to `"Specify your dataset modality (text, tabular, image):"`.
+The metadata file, `labels.csv`, must contain at least three columns:
 
-* Enter `text` to only find label errors based on a single column of text in your dataset.
-* Enter `tabular` to find data and label issues based on any subset of the column features.
-* Enter `image` to find data and label issues in your image dataset.
+- an ID column (with unique identifiers for each datapoint)
+- a path column (with relative paths to image files)
+- a label column (with categorical labels)
 
-To upload a dataset with a schema:
+The metadata file is allowed to have extra columns with various types of metadata.
 
-`cleanlab dataset upload -f [dataset filepath] -s [schema filepath]`
+For example:
```

Comment:
I would move the example labels csv up closer to the example directory format, since they're so closely related.